resources/lib/tubecast/ssdp.py

```diff
@@ -34,7 +34,8 @@ def __init__(self, handler, poll_interval):
         self._handler = handler
 
     def start(self):
-        self._thread = t = threading.Thread(target=self.serve_forever,
+        self._thread = t = threading.Thread(name=type(self).__name__,
```

Comment:
Using the name of the class here because this is in the `ControlMixin` class. Currently the only class deriving from it is "MulticastServer" but if another one happens to be added in the future it shouldn't share the same thread name, hence why I chose to use the class name.